test/sanity/code-smell/docs-build.requirements.in

```
@@ -1,8 +1,9 @@
 jinja2
 pyyaml
 resolvelib < 0.9.0
-sphinx == 4.2.0
+sphinx == 5.1.1
+rstcheck < 6.0.0 # above this causes errors we need to fix in docs
 sphinx-notfound-page
 sphinx-ansible-theme
 straight.plugin
-antsibull-docs == 1.0.0  # currently approved version
+antsibull-docs == 1.3.0  # currently approved version
```

Would be nice to actually use 1.4.0 and not 1.3.0 :)
```suggestion
antsibull-docs == 1.4.0  # currently approved version
```